build/charts/theia/templates/grafana/dashboard-configmap.yaml

```diff
@@ -0,0 +1,7 @@
+apiVersion: v1
+kind: ConfigMap
+metadata:
+  name: grafana-dashboard-config
+  namespace: {{ .Release.Namespace }}
+data:
+{{ (.Files.Glob "provisioning/dashboards/*.json").AsConfig | indent 2}}
```

Comment:
Is that possible to add the dashboard json file names as a parameter into values.yaml? so that users can select on which dashboards they want to include in Grafana

Reply:
Updated.